ci-operator/step-registry/openshift/agentic/trt/review-responder/openshift-agentic-trt-review-responder-commands.sh

```
@@ -0,0 +1,109 @@
+    REVIEWS_JSON=$(echo "${raw_reviews}" | jq --argjson trusted "${trusted_jq_filter}" '[.[] | select(.user.login as $u | $trusted | index($u))]')
+
+    comment_count=$(echo "${COMMENTS_JSON}" | jq 'length')
+    review_count=$(echo "${REVIEWS_JSON}" | jq '[.[] | select(.state != "APPROVED" and .state != "PENDING")] | length')
+    TOTAL=$(( comment_count + review_count ))
+
+    echo "Found ${TOTAL} comment(s)/review(s) from trusted users."
+
+    if [[ "${TOTAL}" -gt 0 ]]; then
+        echo "Addressing review comments..."
+
+        REVIEW_BODY=$(echo "${COMMENTS_JSON}" | jq -r '.[] | "**\(.user.login)** on `\(.path // "general")`:\n\(.body)\n---"' 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] REVIEW_BODY appears unused. Verify use (or export if used externally).

(SC2034)